src/scry/workspace.cr

```diff
@@ -27,7 +27,7 @@ module Scry
         @open_files[file.filename] = {file, Completion::MethodDB.new}
       else
         file_dependencies = @dependency_graph[file.filename].descendants.map &.value
-        method_db = Completion::MethodDB.generate(file_dependencies)
+        method_db = Completion::MethodDB.generate((file_dependencies + [file.filename]).uniq)
```

Comment:
Well, I guess I need to update #92 after merging this :+1: 

https://github.com/crystal-lang-tools/scry/blob/9e89531b108c244df5540db5cb9a9cf1e088fef9/src/scry/workspace.cr#L26-L32